File: app/src/main/java/com/xu/database/Line.java
package com.xu.database;

import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

public class Line {

    private List<MapData> line = new ArrayList<>();

    public Line(MapData... mapData){
        Collections.addAll(line, mapData);
    }

    public List<MapData> get() {
        return line;
    }
}

File: app/src/main/java/com/xu/database/Line.kt
package com.xu.database

import java.util.ArrayList
import java.util.Collections

class Line(vararg mapData: MapData) {

    private val line = ArrayList<MapData>()

    init {
        Collections.addAll(line, *mapData)
    }

    fun get(): List<MapData> {
        return line
    }
}
